Makefile

```diff
@@ -1,9 +1,58 @@
 TOOL_VERSION = 2.1.12
 TEST_DATA = test/simpleTestForGenerator
-
+GIT_USERNAME=License Publisher (maintained by Gary O'Neall)
+GIT_EMAIL=gary@sourceauditor.com
+LICENSE_DATA_REPO=https://github.com/goneall/license-list-data.git
+LICENSE_DATA_REPO_NO_SCHEME=github.com/goneall/license-list-data.git
+LICENSE_DATA_URL=https://$(GITHUB_TOKEN)@$(LICENSE_DATA_REPO_NO_SCHEME)
+LICENSE_OUTPUT_DIR=.tmp
+VERSION=$(shell git describe --always || echo 'UNKNOWN')
+RELEASE_DATE=$(shell date -I)
+COMMIT_MSG="Automatic update of license list data from license XML repository version $(VERSION)"
+#TODO Change the license data repo to license-list-xml before merging pull request
+	
 .PHONY: validate-canonical-match
 validate-canonical-match: licenseListPublisher-$(TOOL_VERSION).jar-valid resources/licenses-full.json $(TEST_DATA) .tmp
-	java -jar -DLocalFsfFreeJson=true -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-$(TOOL_VERSION).jar LicenseRDFAGenerator src .tmp 1.0 2000-01-01 $(TEST_DATA) expected-warnings
+	java -jar -DLocalFsfFreeJson=false -DlistedLicenseSchema="schema/ListedLicense.xsd" licenseListPublisher-$(TOOL_VERSION).jar LicenseRDFAGenerator src .tmp 1.0 2000-01-01 $(TEST_DATA) expected-warnings
+
+.PHONY: deploy-license-data
+deploy-license-data: licenseListPublisher-$(TOOL_VERSION).jar-valid resources/licenses-full.json $(TEST_DATA) .tmp	
```

Comment:
No need for the `resources/licenses-full.json` dependency here, since you set `-DLocalFsfFreeJson=false` below.  Or maybe I'm misunderstanding that option?